### PR TITLE
update lfai and community information in readme

### PR DESCRIPTION
**Pull request by merobi-hub**
### Problem

The main readme is missing a number of our communication channels and says OpenLineage is a sandbox project with the LFAI. It's also missing at least one talk, and the Flink integration is not on the integrations list. 

### Solution

This adds the YouTube channel, wiki, mailing list, website, LinkedIn and Mastodon links to the community section. It also changes "sandbox" to "incubation," adds Julien's Berlin Buzzwords talk in 2021 to the talks section, expands the description of Egeria so it is parallel with the Marquez description, and adds the Flink integration.

> **Note:** All schema changes require discussion. Please [link the issue](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue) for context.

- [ ] Your change modifies the [core](https://github.com/OpenLineage/OpenLineage/blob/main/spec/OpenLineage.json) OpenLineage model
- [ ] Your change modifies one or more OpenLineage [facets](https://github.com/OpenLineage/OpenLineage/tree/main/spec/facets)

If you're contributing a new integration, please specify the scope of the integration and how/where it has been tested (e.g., Apache Spark integration supports `S3` and `GCS` filesystem operations, tested with AWS EMR).

### Checklist

- [x] You've [signed-off](https://github.com/OpenLineage/OpenLineage/blob/main/why-the-dco.md) your work
- [x] Your pull request title follows our [guidelines](https://github.com/OpenLineage/OpenLineage/blob/main/CONTRIBUTING.md#creating-pull-requests)
- [ ] Your changes are accompanied by tests (_if relevant_)
- [x] Your change contains a [small diff](https://kurtisnusbaum.medium.com/stacked-diffs-keeping-phabricator-diffs-small-d9964f4dcfa6) and is self-contained
- [ ] You've updated any relevant documentation (_if relevant_)
- [ ] Your comment includes a one-liner for the changelog about the specific purpose of the change (_if necessary_)
- [ ] You've versioned the core OpenLineage model or facets according to [SchemaVer](https://docs.snowplowanalytics.com/docs/pipeline-components-and-applications/iglu/common-architecture/schemaver) (_if relevant_)
- [ ] You've added a [header](https://github.com/OpenLineage/OpenLineage/tree/main/.github/header_templates.md) to source files (_if relevant_)

----
SPDX-License-Identifier: Apache-2.0\
Copyright 2018-2023 contributors to the OpenLineage project

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -62,22 +62,29 @@ The OpenLineage repository contains integrations with several systems.
...
 
 ## Talks
- - [Metadata day 2020. Open Lineage lightning talk](https://www.youtube.com/watch?v=anlV5Er_BpM)
+ - [Metadata Day 2020. Open Lineage lightning talk](https://www.youtube.com/watch?v=anlV5Er_BpM)
  - [Open Core Summit 2020. Observability for data pipelines. OpenLineage project launch](https://www.coss.community/coss/ocs-2020-breakout-julien-le-dem-3eh4)
- - [Data Driven NYC Feb 2021. Data Observability and Pipelines: OpenLineage and Marquez](https://mattturck.com/datakin/)
- - [Big Data Technology Warsaw Summit. Data lineage and Observability with Marquez and OpenLineage](https://conference.bigdatatechwarsaw.eu/julien-le-dem/)
+ - [Data Driven NYC February 2021. Data Observability and Pipelines: OpenLineage and Marquez](https://mattturck.com/datakin/)
+ - [Berlin Buzzwords June 2021. Observability for Data Pipelines with OpenLineage](https://2021.berlinbuzzwords.de/member/julien-le-dem)
+ - [Big Data Technology Warsaw Summit 2023. Data lineage and Observability with Marquez and OpenLineage](https://bigdatatechwarsaw.eu/prelegenci/julien-le-dem/)
```
Comment (mobuchowski) [inline README.md:87]: I think this link is wrong, also, Julien spoke there in 2021.

File: README.md [hunk 2]
```diff
@@ -62,22 +62,29 @@ The OpenLineage repository contains integrations with several systems.
...
-- Google group: [openlineage@googlegroups.com](https://groups.google.com/g/openlineage)
+- Mailing list: [openlineage-tsc](https://lists.lfaidata.foundation/g/openlineage-tsc)
+- Wiki: [OpenLineage+Home](https://wiki.lfaidata.foundation/display/OpenLineage/OpenLineage+Home)
+- LinkedIn: [13927795](https://www.linkedin.com/groups/13927795/)
+- YouTube: [channel](https://www.youtube.com/channel/UCRMLy4AaSw_ka-gNV9nl7VQ)
+- Mastodon: [@openlineage@fostodon.org](openlineage@fosstodon.org)
+- Website: [openlineage.io](http://openlineage.io)
 
 ## Talks
- - [Metadata day 2020. Open Lineage lightning talk](https://www.youtube.com/watch?v=anlV5Er_BpM)
```
Comment (mobuchowski) [inline README.md:77]: What do you think about having newest talks at the top?

File: README.md [hunk 3]
```diff
@@ -62,22 +62,29 @@ The OpenLineage repository contains integrations with several systems.
...
+ - [Metadata Day 2020. Open Lineage lightning talk](https://www.youtube.com/watch?v=anlV5Er_BpM)
  - [Open Core Summit 2020. Observability for data pipelines. OpenLineage project launch](https://www.coss.community/coss/ocs-2020-breakout-julien-le-dem-3eh4)
- - [Data Driven NYC Feb 2021. Data Observability and Pipelines: OpenLineage and Marquez](https://mattturck.com/datakin/)
- - [Big Data Technology Warsaw Summit. Data lineage and Observability with Marquez and OpenLineage](https://conference.bigdatatechwarsaw.eu/julien-le-dem/)
+ - [Data Driven NYC February 2021. Data Observability and Pipelines: OpenLineage and Marquez](https://mattturck.com/datakin/)
+ - [Berlin Buzzwords June 2021. Observability for Data Pipelines with OpenLineage](https://2021.berlinbuzzwords.de/member/julien-le-dem)
```
Comment (mobuchowski) [inline README.md:86]: Maybe link 2022 talk? https://www.youtube.com/watch?v=pLBVGIPuwEo

File: README.md (hunk 1 above)
Comment (merobi-hub) [reply README.md:87]: Thanks @mobuchowski . The link was broken, and this is one I could find (and it was dated incorrectly, I guess). I'll look for another one and fix the year.

**Review by mobuchowski — APPROVED** [review]

**Pull request merged**